vendor/engines/c2po/spec/models/credit_card_account_spec.rb

```diff
@@ -37,7 +37,7 @@
   end
 
   it "should be limited to a single facility" do
-    expect(CreditCardAccount.limited_to_single_facility?).to be true
+    expect(CreditCardAccount.single_facility?).to be true
```

Comment:
Sounds good, this got changed with a search/replace.  But I can improve the code since I'm touching it anways.